platipy/imaging/dose/dvh.py

```diff
@@ -49,8 +49,8 @@ def calculate_dvh(dose_grid, label, bins=1001):
     values = np.cumsum(counts[::-1])[::-1]
     if np.all(values == 0):
         return bins, values
-    else:
```

Comment:
This will return an error if there is zero dose to a particular structure. The else statement fixes this.

Reply:
Hey Rob, the reason I removed the else here is that if the if statement on line 50 is true, the function returns on line 51 and will never proceed to line 53. This was a pylint recommendation to remove the else since the functionality is the same either way.

Reply:
Ah yes I understand, good pick up!